packages/namers/default/src/DefaultNamer.js

```diff
@@ -9,6 +9,8 @@ import path from 'path';
 
 const COMMON_NAMES = new Set(['index', 'src', 'lib']);
 
+const namedTargets: Set<Target> = new Set();
```

Comment:
Hmm this seems... interesting. Why?

Reply:
A bit of a hack to avoid naming more than one bundle as the dist entry. The case here was an html entry file linking to another html file, which also got tagged as an entry in the same bundle group to avoid being tagged with a digest.

Arguably in this case, that other html file shouldn't be part of the same bundle group?

Reply:
Hmm yes. How did multiple html files get added to the same bundle group? Aren't all HTML dependencies async?

Reply:
resolved :) `addURLDependency` makes sure of this now.